### PR TITLE
Run E2E in Github Actions 

### PR DESCRIPTION
**Pull request by joan41868**
This **PR** provides github actions job for running E2E suites against the current branch.

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+import com.hedera.services.bdd.suites.token.TokenTransactSpecs;
+import com.hedera.services.bdd.suites.token.TokenUpdateSpecs;
+import com.hedera.services.bdd.suites.token.UniqueTokenManagementSpecs;
+
+// TODO: Refactor to better name
+// TODO: Remove stress tests if confirmed
+// TODO: Include performance tests if confirmed
+
+// TODO: Verify in daily to run sync() or async()
+// TODO: Implement a solution, extracting only failing test as console output
```
Comment (georgiyazovaliiski) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:187]: Is this implemented yet?

Comment (joan41868) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:187]: I'm waiting for @CordonaCodeCraft to finish his work on the maven profile. Those should have already been removed by that time.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java [hunk 2]
```diff
@@ -0,0 +1,482 @@
+package com.hedera.services.bdd.suites;
+
```
Comment (Daniel-K-Ivanov) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:2]: License is missing

File: test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+import com.hedera.services.bdd.suites.token.TokenManagementSpecsStateful;
+import com.hedera.services.bdd.suites.token.TokenMiscOps;
+import com.hedera.services.bdd.suites.token.TokenPauseSpecs;
+import com.hedera.services.bdd.suites.token.TokenTotalSupplyAfterMintBurnWipeSuite;
+import com.hedera.services.bdd.suites.token.TokenTransactSpecs;
+import com.hedera.services.bdd.suites.token.TokenUpdateSpecs;
+import com.hedera.services.bdd.suites.token.UniqueTokenManagementSpecs;
+
+// TODO: Refactor to better name
+// TODO: Remove stress tests if confirmed
```
Comment (Daniel-K-Ivanov) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:183]: Yes, we should remove stress tests

File: test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+// TODO: Include performance tests if confirmed
```
Comment (Daniel-K-Ivanov) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:184]: We should not include perf tests

File: test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java [hunk 5]
```diff
@@ -0,0 +1,482 @@
...
+
+// TODO: Verify in daily to run sync() or async()
```
Comment (Daniel-K-Ivanov) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java:186]: Each test suite specifies the way it should run (sync/async)

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]
For this PR to be merged and sent for review, do we need to get the @CordonaCodeCraft PR ready and merged into this branch?

**Comment by stoqnkpL** [comment]
This logic will be merged as part of #826

**Pull request closed without merging**